core-bundle/src/Resources/contao/classes/Backend.php

```diff
@@ -697,7 +697,7 @@ public static function addPageIcon($row, $label, DataContainer $dc=null, $imageA
 		}
 
 		$image = Controller::getPageStatusIcon((object) $row);
-		$imageAttribute = trim($imageAttribute . ' data-icon="' . Image::getPath(Controller::getPageStatusIcon((object) array_merge($row, array('published'=>'1')))) . '" data-icon-disabled="' . Image::getPath(Controller::getPageStatusIcon((object) array_merge($row, array('published'=>'')))) . '"');
+		$imageAttribute = trim($imageAttribute . ' data-icon="' . Image::getPath(Controller::getPageStatusIcon((object) array_merge($row, array('published'=>1)))) . '" data-icon-disabled="' . Image::getPath(Controller::getPageStatusIcon((object) array_merge($row, array('published'=>0)))) . '"');
```

Comment:
Shouldn‘t this be `true` here as well?

Reply:
The `$row` is a `\Contao\Database\Result` row and not the row of a Model.